test/init.js

```diff
@@ -0,0 +1,23 @@
+import jsdom from 'jsdom';
+
+// `window` must exist before including
+// the polyfill, which gets reset between tests,
+// and also before calling CustomEvent
+global.window = jsdom.jsdom().defaultView;
+global.document = global.window.document;
+global.navigator = global.window.navigator;
+
+// cardboard-vr-display uses CustomEvent globals; shim
+// so this runs in node
+// Logic similar to https://github.com/webmodules/custom-event
+// but can't rely on it's feature detection due to timing
+// of including the shim and populating window/document
+global.CustomEvent = function CustomEvent (type, params) {

```

Comment:
no space after CustomEvent

Reply:
Also, why not use (type, params) => {

Seems clearer...

Reply:
lint fixed, and this is a constructor to simulate the global in cardboard-vr-display (which relies on globals unfortunately) as these run in a node environment